README.md

```diff
@@ -16,10 +16,38 @@ This extension adds language support for the Nim language to VS Code, including:
 - Build-on-save
 - Workspace symbol search
 - Quick info
-- Nim check result reported in `Nim` output channel (great for macro development).
 - Problem Matchers for nim compiler and test output
+- Nim check result reported in `Nim` output channel (great for macro development) <details><summary>demo</summary>![output channel demo](images/nim_vscode_output_demo.gif "Demo of macro evaluation in the output channel")</details>
 
-![output channel demo](images/nim_vscode_output_demo.gif)
+[Read how it differs](#history--trivia) from other Nim extensions, of which there are a surprising number.
+
+## History & Trivia
+
+This extension started out as a [Nim port](https://github.com/pragmagic/vscode-nim/issues/172) of an extension written in [TypeScript](https://marketplace.visualstudio.com/items?itemName=kosz78.nim) for the Nim language. The port was done as a learning exercise for myself (@saem) and also meant to allow the community to contribute to it more easily. For the "full museum" experience be sure to checkout the [original announcement](https://forum.nim-lang.org/t/6862#42899) on the nim forums. This entire section wouldn't be here if it wasn't for [cpbotha's post suggesting it](https://forum.nim-lang.org/t/7242#45847), thanks.
+
+Since then community members (noteably: @RSDuck, @dscottboggs, and yours truly) have made various contributions directly to this version. From bug fixes to small enhancements.
```

Comment:
+1 for the usage of yours truly

Reply:
Thank you, thank you.

I prefer a bit of personality and don't want this to feel like work.